home-page.html

```diff
@@ -11,16 +11,69 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+    <script src="script.js" defer></script>
+    <script src="contributors.js" defer></script>
+    <link rel="stylesheet" href="pop-up-menu.css">
 </head>
-<body>
+<body id="home-page">
+ <div class="general-grid">
     <section class="background-image-wrapper">
         <div class="over-lay">
-            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i></header>
+            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i>
+                <div id="pop-up-menu-wrapper">
+                    <i class="fa-solid fa-xmark" id="cancel"></i>
+                <div class="pop-up-menu">
+                    <ul class=".pop-up-list">
+                        <li class="pop-up-li"><a class="page-link" href="home-page.html">Home</a></li>
+                        <li class="pop-up-li"><a class="page-link" href="index.html">About</a></li>
+                        <li class="pop-up-li">Program</li>
+                        <li class="pop-up-li">Join</li>
+                        <li class="pop-up-li">Sponsor</li>
+                    </ul>
+                </div>
+                </div></header>
+            <header class="about-desk-head">
+                <div class="sign-in-bar">
+                    <i class="fa-brands fa-facebook"></i>
+                    <i class="fa-brands fa-twitter"></i>
+                    <ul>
+                        <li><a href="">Logout</a></li>
+                        <li><a href="">Mypage</a></li>
+                        <li><a href="">Extras</a></li>
+                    </ul>
+                </div>
```

Comment:
- [ ] Kindly, consider using a `nav` tag at sign-in-bar. This will improve the sementic of the page 👍 🎖️